fvm/src/syscalls/vm.rs

```diff
@@ -49,27 +49,5 @@ pub fn abort(
 }
 
 pub fn context(context: Context<'_, impl Kernel>) -> crate::kernel::Result<InvocationContext> {
-    use anyhow::Context as _;
-
-    Ok(InvocationContext {
-        caller: context.kernel.msg_caller(),
```

Comment:
we should probably remove all these random ops from the kernel while we're at it.

Reply:
hmm, this was more in the direction of splitting context method into smaller structs defined in their own relative modules e.g. `NetworkContext`, `MessageContext` rather than removing ops entirely.

do you mean to remove these syscalls entirely and just use `.x_context` as the way to read them?

Reply:
I mean removing these kernel methods, specifically, and just sticking with the `x_context` methods. Yes.

Unless something is using them. It makes it easier to tell what's "relevant" and what's not.

Reply:
Doing this in a different PR since its a bit out of scope for this one.